### ISSUE TITLE
[4.0] Prevent sticky toolbar jump on page scroll

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#15989]
### Steps to reproduce the issue

This jump happed after the top headers were merged into 1.
Think there is a set `calc()` value which needs to be changed, but the value needs to be more dynamic.

**Comment by ciar4n** [comment]
It should be just a case of changing this back to 0.. https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/js/template.js#L320

I will take a look once #15937 is merged.

**Comment by ciar4n** [comment]
PR created... #16018
